File: ipman.shared/Entity/FaceImage.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ipman.shared.Entity
{
    public class FaceImage : EntityBase
    {
        public Guid FaceCollectionID { get; set; }
        public FaceCollection FaceCollection {get; set;}
        public string BlobPath { get; set; }
        public DateTime CreatedUTC { get; set; }
        public bool IsActive { get; set; }
    }
}

File: ipman/ClientApp/entity/FaceImage.ts
import { EntityBase } from "./EntityBase";
import { FaceCollection } from "@entity";

export default interface FaceImage extends EntityBase{
    FaceCollectionID: string;
    FaceCollection: FaceCollection;
    BlobPath: string;
    CreatedUTC: Date | string;
    IsActive: boolean;
}